internal/lockclient/simple_client.go

```diff
@@ -24,18 +30,60 @@ var _ Client = (*SimpleClient)(nil)
 
 // SimpleClient implements Client, the lockclient for Lockey.
 type SimpleClient struct {
+	Id     uint8
```

Comment:
What ID is this? Is it being used?

Reply:
Not yet, I guess it can be omitted for this version
It's for the "uniqueness" that we discussed